### ISSUE TITLE
About Pretrained model

### ISSUE THREAD

**Issue by Uriellll22** [weepiess/StyleFlow-Content-Fixed-I2I#6]
Hi! Your works are so impressive! I'm looking forward to the pre-trained model for GTA->Cityscapes! Thank you!

**Comment by WeichenFan** [comment]
> Hi! Your works are so impressive! I'm looking forward to the pre-trained model for GTA->Cityscapes! Thank you!

Hi, thank you for your interest :)

We just updated GTA -> CItyscapes with tv loss in 300x400, and more settings will come soon. Feel free to use :)

**Comment by Uriellll22** [comment]
It's so helpful for me. Thanks a lot！
